frontend/packages/dev-console/integration-tests/features/topology/topology-smoke_test.feature

```diff
@@ -0,0 +1,165 @@
+Feature: Topology smoke test
+	As a user, I want to verify topology feature    
+
+@regression, @smoke
+Scenario: Empty state of topology
+   Given user is in administrative perspective
+   When user goes to developer perspective
+   And user creates a new project
+   Then user sees Add page with message on the top "No workloads found"
+
+@regression, @smoke
+Scenario: Topology with workloads
+   Given user is in administrative perspective
+   When user goes to developer perspective
+   And user selects a existing project from project list with existing workloads
+   Then user sees different workloads in topology chart area
+
+@regression, @smoke
```

Comment:
This the visual interpretation and hard to automate it. Can you please add @manual tag here